draft-ietf-rtcweb-jsep.xml

```diff
@@ -446,7 +446,9 @@ setRemote(ANSWER) |                                   |
           stopped and disassociated from m= sections. An RtpTransceiver
           is never associated with more than one m= section, and once a
           session description is applied, a m= section is always
-          associated with exactly one RtpTransceiver.</t>
+          associated with exactly one RtpTransceiver. An RtpTransceiver
```

Comment:
I would move this sentence before the previous sentence, so that you can explain what association is before you provide additional detail on what it isn't.